controllers/app.controller.js

```diff
@@ -9,3 +11,9 @@ exports.getTopic = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getApi = (req, res) => {
+  const endpointsFilePath = path.join(__dirname, "..", "endpoints.json");
+  const endpointsData = JSON.parse(fs.readFileSync(endpointsFilePath, "utf8"));
```

Comment:
If you are doing any data reading fetching, formatting etc, this should be done with a model. 
However arguably, we could just require in the endpoints.json in the controller and send!   Take your pic!